install.sh

```diff
@@ -41,8 +41,8 @@ sed -i ${txt} "s@^export JAVA_HOME=.*@export JAVA_HOME=${javaHome}@g" conf/env/d
 # Change configuration in conf/datasource.properties
 sed -i ${txt} "s@^spring.datasource.driver-class-name=.*@spring.datasource.driver-class-name=${datasourceDriverClassname}@g" conf/datasource.properties
 sed -i ${txt} "s@^spring.datasource.url=.*@spring.datasource.url=jdbc:${dbtype}://${dbhost}/${dbname}?characterEncoding=UTF-8\&allowMultiQueries=true@g" conf/datasource.properties
-sed -i ${txt} "s@^spring.datasource.username=.*@spring.datasource.username=${username}@g" conf/datasource.properties
-sed -i ${txt} "s@^spring.datasource.password=.*@spring.datasource.password=${password}@g" conf/datasource.properties
+sed -i ${txt} "s/^spring.datasource.username=.*/spring.datasource.username=${username}/g" conf/datasource.properties
+sed -i ${txt} "s/^spring.datasource.password=.*/spring.datasource.password=${password}/g" conf/datasource.properties
```

Comment:
Good catch! But when we password contains `/`, does this command work?

Reply:
I think maybe we should find more general way to escape specific characters

Reply:
Yeah，need to consider both of  's','g','@'

Reply:
Yes, we have to consider other character that must be escaped. maybe we should add a escape function to do that, WDYT? @tracehh

Reply:
Yes，I am working on this